packages/InteractiveProfilingTool-Core.package/IPTModel.class/instance/buildWindowWith..st

```diff
@@ -0,0 +1,11 @@
+user interface
+buildWindowWith: aBuilder
+	"Basis Window. Stops running profiling when closed."
```

Comment:
While I’m personally in favor of more comments instead of less, the majority opinion in the Squeak community seems to be to treat comments as an exception… So I guess the people grading us would consider this comment redundant.